frontend/public/components/factory/table-filters.ts

```diff
@@ -100,12 +100,7 @@ export const tableFilters: TableFilterMap = {
   'role-binding-roleRef-kind': (kind: string, binding) => binding.roleRef.kind === kind,
 
   // Filter role bindings by user name
-  'role-binding-user': (userName, binding) =>
-    _.some(binding.subjects, {
-      kind: 'User',
-      apiGroup: 'rbac.authorization.k8s.io',
-      name: userName,
-    }),
+  'role-binding-user': (userName, { subject }) => subject.name === userName,
```

Comment:
@cyril-ui-developer do you know if `subjects` array in the original `binding` object will ever contain more than one array item? Wondering if we should display a binding that contains the `userName` in the array of `subjects` even if other subjects are present? (but not those bindings that do not contain the user name).

![image](https://user-images.githubusercontent.com/21317056/116271661-2f6ea480-a74e-11eb-8258-725c6c7c211d.png)